sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -919,6 +919,8 @@ object FunctionRegistry {
   registerInternalExpression[EWM]("ewm")
   registerInternalExpression[NullIndex]("null_index")
   registerInternalExpression[CastTimestampNTZToLong]("timestamp_ntz_to_long")
+  registerInternalExpression[FromProtobuf]("from_protobuf")
```

Comment:
You can either use the Session extensions, or use this. Which one works?

Reply:
It seems like the Session extensions is not working from my PR for some reason, because `(from|to)_protobuf` are not working when I commented the lines here.

Reply:
Confirmed that Protobuf functions are working fine with the `registerInternalExpression` here. Let me use this if they are working as the same.

Reply:
Same as https://github.com/apache/spark/pull/47885#discussion_r1743595767.